components/clp-package-utils/clp_package_utils/scripts/native/archive_manager.py

```
@@ -34,10 +34,10 @@
 
 
 class DeleteHandler(ABC):
-    def __init__(self, query_params: List[str]):
-        self._params: List[str] = query_params
+    def __init__(self, query_params: list[str]):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)